src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
```

Comment:
According to the checklist, there should not be an empty line at the beginning of the class definition.